packages/breadcrumb/components/Breadcrumb.tsx

```diff
@@ -9,8 +18,34 @@ export interface BreadcrumbProps {
 class Breadcrumb extends React.PureComponent<BreadcrumbProps, {}> {
   public render() {
     const { children } = this.props;
+    const breadcrumbSeparator = (
+      <Icon shape={SystemIcons.CaretRight} size={iconSizeXs} />
+    );
+    const crumbsArr = React.Children.toArray(children).reduce(
+      (acc, crumb, i) => {
+        acc.push(crumb);
+        if (i !== React.Children.toArray(children).length - 1) {
+          acc.push(breadcrumbSeparator);
+        }
+        return acc;
+      },
+      Array()
```

Comment:
is there a technical reason for using `Array()` here instead of `[]` or is it just personal preference?